lerna.json

```diff
@@ -5,12 +5,14 @@
     }
   },
   "npmClient": "yarn",
-  "packages": ["packages/*"],
```

Comment:
Just a comment - but I think I see the formatting of this line change in every PR. What rule should we enable to make this consistent?

Reply:
I think this is lerna modifying it vs prettier formatting it. Not sure what to do at the moment to fix it, but good to flag it